datafusion/core/src/execution/context.rs

```diff
@@ -3322,173 +3334,6 @@ mod tests {
         Ok(())
     }
 
-    #[tokio::test]
```

Comment:
Moved out of `context.rs` and into `datafusion/core/tests/sql/idenfifers.rs` without change